doc/Manuals/ug-syspar_x-road_v6_system_parameters.md

```diff
@@ -276,14 +276,7 @@ limits the configuration parameters usage:
 This in practice limits configurability to different kinds of PostgreSQL database configurations, and it is not possible
 to use for example MySQL as a data store for REST API module.
 
-### 3.3 Proxy User Interface parameters: `[proxy-ui]`
-
-| **Parameter**                                    | **Vanilla value**                          | **Description** |
-|--------------------------------------------------|--------------------------------------------|-----------------|
-| wsdl-validator-command                           |                                            | The command to validate the given X-Road service WSDL. The command script must:<br/>a) read the WSDL from the URI given as an argument,<br/>b) return exit code 0 on success,<br/>c) return exit code 0 and write warnings to the standard error (*stderr*), if warnings occurs,<br/>d) return exit code other then 0 and write error messages to the standard error (*stderr*), if errors occurs.<br/>Defaults to no operation. |
-| auth-cert-reg-signature-digest-algorithm-id      | SHA-512                                    | Signature digest algorithm used for generating authentication certificate registration request.<br/>Possible values are<br/>-   SHA-256,<br/>-   SHA-384,<br/>-   SHA-512. |
-
-### 3.4 Signer parameters: `[signer]`
+### 3.3 Signer parameters: `[signer]`
```

Comment:
Maybe it would be safer to keep the numbering (just leave an empty 3.3 section "proxy-ui removed in version 6.24" or something) in case there are some external links to the parameter sections.

Reply:
Fixed